### PR TITLE
[14.0][MIG] sale_stock_secondary_unit: Migration to 14.0 

### PR DESCRIPTION
**Pull request by toita86**

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

File: sale_stock_secondary_unit/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Sale Stock Secondary Unit",
     "summary": "Get product quantities in a secondary unit",
-    "version": "15.0.1.0.2",
+    "version": "14.0.1.0.2",
```
Comment (geomer198) [inline sale_stock_secondary_unit/__manifest__.py]: I think module version must be `14.0.1.0.0`

**Review by geomer198 — COMMENTED** [review]
Please check my comment.

File: sale_stock_secondary_unit/__manifest__.py (hunk 1 above)
Comment (toita86) [reply sale_stock_secondary_unit/__manifest__.py]: Thank you! Just fixed.

**Comment by francesco-ooops** [comment]
@geomer198 good now?

**Review by GSLabIt — COMMENTED** [review]
did u back port from v 15.0?

**Comment by toita86** [comment]
> did u back port from v 15.0?

Yes

**Review by geomer198 — APPROVED** [review]
LGTM

**Comment by GSLabIt** [comment]
@pedrobaeza an info on migration best pratics; it is ok to back port from 15.0 instead of migrating from 13.0?

**Comment by pedrobaeza** [comment]
Well, if there's no feature difference, it doesn't matter too much. What I would advise is to migrate from the closest one in terms of datamodel / framework difference. In this case, it would be the same more or less, so backporting from 15 may be other option.

**Review by GSLabIt — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@pedrobaeza shall we proceed?

**Pull request merged**